src/main/kotlin/uk/gov/dwp/dataworks/Model.kt

```diff
@@ -43,3 +47,67 @@ data class UserTask(val correlationId: String,
         }
     }
 }
+
+@JsonDeserialize(builder = StatementObject.StatementObjectBuilder::class)
+data class StatementObject(var Sid: String, var Effect: String, var Action: List<String>, var Resource: List<String>) {
+    @JsonPOJOBuilder(buildMethodName = "StatementObjectBuilder", withPrefix = "set")
+    class StatementObjectBuilder {
+        private lateinit var Sid: String
+        private lateinit var Effect: String
+        private lateinit var Action: List<String>
+        private lateinit var Resource: List<String>
+
+        @JsonProperty("Sid")
+        fun setSid(Sid: String): StatementObjectBuilder {
+            this.Sid = Sid
+            return this
+        }
+
+        @JsonProperty("Effect")
+        fun setEffect(Effect: String): StatementObjectBuilder {
+            this.Effect = Effect
+            return this
+        }
+
+        @JsonProperty("Action")
+        fun setAction(Action: List<String>): StatementObjectBuilder {
+            this.Action = Action
+            return this
+        }
+
+        @JsonProperty("Resource")
+        fun setResource(Resource: List<String>): StatementObjectBuilder {
+            this.Resource = Resource
+            return this
+        }
+
+        fun StatementObjectBuilder(): StatementObject {
+            return StatementObject(Sid, Effect, Action, Resource)
+        }
+    }
+}
+
+@JsonDeserialize(builder = JsonObject.JsonObjectBuilder::class)
+data class JsonObject( var Version: String, var Statement: List<StatementObject>){
```

Comment:
This doesn't look like a general JSON Object, more like a `AwsIamPolicyJsonObject`